<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>1191</width>
         <height>691</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,5">
        <item>
         <widget class="QFrame" name="frBarraLateral">
          <property name="styleSheet">
           <string notr="true">
QFrame{
    background-color: rgb(62, 159, 151);
}
QPushButton{
    background-color: rgb(61, 61, 61);
    border-radius: 10px;
    color: rgb(255, 255, 255);
    font: 77 10pt &quot;Arial Black&quot;;
padding:2px;
}

QPushButton:hover{
    background-color: white;
    color: rgb(0, 0, 0);
    font: 77 10pt &quot;Arial Black&quot;;
}

QPushButton#customButton{
    background-color: rgb(62, 159, 151);
}
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QFrame" name="verticalFrame_4">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>80</y>
             <width>181</width>
             <height>521</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="btnFilasColumnas">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Filas - Columnas</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSeleccionar">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Seleccionar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnResultado">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Resultado</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTablero">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Tablero</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnPrediccion">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Prediccion</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="btnCargarArchivo">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>650</y>
             <width>158</width>
             <height>24</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Cargar Archivo</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frPaginas">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>971</width>
             <height>681</height>
            </rect>
           </property>
           <widget class="QWidget" name="pageCargarData">
            <widget class="QLabel" name="lblTitulo">
             <property name="geometry">
              <rect>
               <x>340</x>
               <y>30</y>
               <width>311</width>
               <height>61</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">qproperty-alignment: 'AlignCenter';
</string>
             </property>
             <property name="text">
              <string>BIENVENIDO</string>
             </property>
            </widget>
            <widget class="QFrame" name="frCargar">
             <property name="geometry">
              <rect>
               <x>280</x>
               <y>180</y>
               <width>401</width>
               <height>241</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
    background-color: #303030;
    border-radius: 5px;
    color: rgb(255, 255, 255);
    font: 77 10pt &quot;Arial Black&quot;;
padding:6px;
}

QPushButton:hover{
    background-color: white;
    color: rgb(0, 0, 0);
    font: 77 10pt &quot;Arial Black&quot;;
}


</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QLabel" name="lblCargando">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>160</y>
                <width>401</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">qproperty-alignment: 'AlignCenter';
</string>
              </property>
              <property name="text">
               <string>Cargando...</string>
              </property>
             </widget>
            </widget>
           </widget>
           <widget class="QWidget" name="pageFilasColumnas">
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>20</y>
               <width>981</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">qproperty-alignment: 'AlignCenter';</string>
             </property>
             <property name="text">
              <string>Informacion del archivo</string>
             </property>
            </widget>
            <widget class="QLabel" name="lblNFilas">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>160</y>
               <width>141</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="lblNColumnas">
             <property name="geometry">
              <rect>
               <x>620</x>
               <y>150</y>
               <width>131</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>220</y>
               <width>551</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">qproperty-alignment: 'AlignCenter';
</string>
             </property>
             <property name="text">
              <string>Tipos de Datos</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>300</x>
               <y>320</y>
               <width>55</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
            <widget class="QLabel" name="lblTipoDato">
             <property name="geometry">
              <rect>
               <x>204</x>
               <y>320</y>
               <width>541</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">qproperty-alignment: 'AlignCenter';

</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QFrame" name="frTiposDatos">
             <property name="geometry">
              <rect>
               <x>199</x>
               <y>290</y>
               <width>551</width>
               <height>341</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    qproperty-alignment: 'AlignCenter';
}

QLabel {
    font: 9pt &quot;MS Shell Dlg 2&quot;;
}
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="pageSeleccionar">
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>-10</x>
               <y>10</y>
               <width>981</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">qproperty-alignment: 'AlignCenter';</string>
             </property>
             <property name="text">
              <string>COLUMNAS</string>
             </property>
            </widget>
            <widget class="QFrame" name="frCheckboxes">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>80</y>
               <width>591</width>
               <height>481</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
            <widget class="QPushButton" name="btnSeleccionarChecks">
             <property name="geometry">
              <rect>
               <x>410</x>
               <y>610</y>
               <width>158</width>
               <height>24</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 255, 255);</string>
             </property>
             <property name="text">
              <string>Seleccionar</string>
             </property>
            </widget>
            <widget class="QFrame" name="frSeleccionar">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>80</y>
               <width>601</width>
               <height>481</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
    background-color: rgb(127, 127, 127);
    border-radius: 5px;
    color: rgb(255, 255, 255);
padding:2px;
}

QPushButton:hover{
    background-color: white;
    color: rgb(0, 0, 0);
}
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QPushButton" name="btnSeleccionarPredictora">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>120</y>
                <width>321</width>
                <height>71</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Seleccionar Variable Predictora</string>
              </property>
             </widget>
             <widget class="QPushButton" name="btnSeleccionarIndependientes">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>300</y>
                <width>321</width>
                <height>71</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Seleccionar Variables independientes</string>
              </property>
             </widget>
            </widget>
           </widget>
           <widget class="QWidget" name="pageResultado">
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>-10</x>
               <y>20</y>
               <width>981</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">qproperty-alignment: 'AlignCenter';</string>
             </property>
             <property name="text">
              <string>ANALISIS EXPLORATORIO</string>
             </property>
            </widget>
            <widget class="QWidget" name="gridLayoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>60</y>
               <width>971</width>
               <height>631</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1" columnstretch="0,0">
              <item row="0" column="0" colspan="2">
               <widget class="QFrame" name="frame_2">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <widget class="QLabel" name="lblCuatroFilas">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>50</y>
                   <width>951</width>
                   <height>131</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>6</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style: solid;
border-width: 1px;

border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));


</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
                <widget class="QLabel" name="label_2">
                 <property name="geometry">
                  <rect>
                   <x>4</x>
                   <y>10</y>
                   <width>961</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">qproperty-alignment: 'AlignCenter';
</string>
                 </property>
                 <property name="text">
                  <string>Cinco Primeras Filas</string>
                 </property>
                </widget>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QFrame" name="frame_3">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <widget class="QLabel" name="label_3">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>961</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">qproperty-alignment: 'AlignCenter';
</string>
                 </property>
                 <property name="text">
                  <string>Valores Nulos</string>
                 </property>
                </widget>
                <widget class="QLabel" name="lblCamposNulos">
                 <property name="geometry">
                  <rect>
                   <x>14</x>
                   <y>29</y>
                   <width>951</width>
                   <height>171</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>6</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style: solid;
border-width: 1px;

border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QFrame" name="frame_4">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <widget class="QLabel" name="label_8">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>961</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">qproperty-alignment: 'AlignCenter';
</string>
                 </property>
                 <property name="text">
                  <string>Resumen Estadistico</string>
                 </property>
                </widget>
                <widget class="QLabel" name="lblResumenEstadistico">
                 <property name="geometry">
                  <rect>
                   <x>4</x>
                   <y>40</y>
                   <width>961</width>
                   <height>151</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>7</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style: solid;
border-width: 1px;

border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="pageTablero"/>
           <widget class="QWidget" name="pagePrediccion">
            <widget class="QLabel" name="label_9">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>20</y>
               <width>981</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">qproperty-alignment: 'AlignCenter';</string>
             </property>
             <property name="text">
              <string>Prediccion</string>
             </property>
            </widget>
            <widget class="QLabel" name="lblPrediccion">
             <property name="geometry">
              <rect>
               <x>14</x>
               <y>80</y>
               <width>931</width>
               <height>551</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
